app/src/main/res/layout/dialog_results.xml

```diff
@@ -343,6 +343,7 @@
 
         <include layout="@layout/expanded_results_box"
             android:id="@+id/expandedResultsBox"
+            android:visibility="gone"
```

Comment:
But this seems insufficient—shouldn't we also change the arrow in `resultsMoreInformationArrow`? Amazingly, that seems the only needed change, the code doesn't need further updates.